lib/big_query/timestamp_rollup.ex

```diff
@@ -4,6 +4,7 @@ defmodule BigQuery.TimestampRollup do
   @callback is_a?(String.t) :: boolean
   @callback floor(%DateTime{}) :: %DateTime{}
   @callback ceiling(%DateTime{}) :: %DateTime{}
-  @callback range(%DateTime{}, %DateTime{}, boolean) :: [%DateTime{}]
```

Comment:
Used to support that boolean param, to indicate inclusive-vs-exclusive ranges.  But it was too confusing to have both floating around in the code.